### ISSUE TITLE
Tab switch crashes the site

### ISSUE THREAD

**Issue by dominikgeissler** [sbsev/svelte-site#64]
Just found a really weird bug.
1. Go to "Satzung"
2. Go to "Datenschutz"

The TOC is still shown on the "Datenschutz"-tab. Switching from this tab is impossible (unless reloading).

![grafik](https://user-images.githubusercontent.com/39053640/179393330-11a6f5a2-0aa4-4253-87f3-e5a842baef18.png)

![website-interaction](https://user-images.githubusercontent.com/39053640/179393337-f7dec86d-4f3d-4f0c-9b50-f4e0e2267bff.gif)

**Comment by janosh** [comment]
Weird, I don't see the same behavior in Brave on Mac. But it may very well be that my [ToC component](https://github.com/janosh/svelte-toc) is buggy. Feel free to remove it or replace it with another if necessary.

@dominikgeissler Do you see a console error on the Datenschutz page where navigation no longer works? If so, please paste it here. Also, when you say both pages show the ToC you mean the entries don't update, right? The ToC remains identical?

**Comment by dominikgeissler** [comment]
Nope, no console error (except when trying to interact with the content of the site, then I'm getting the following error message "Uncaught TypeError: E.scrollIntoViewIfNeeded is not a function" in BasePage.svelte). The http-request is sent to the corresponding page when trying to switch tabs, but it is not loaded.
The ToC is the same as from the "Satzung"-tab (and. I can interact with it even when its in the "Datenschutz"-tab, here the anchors are just added to the /datenschutz-url)

**Comment by janosh** [comment]
@dominikgeissler Ah, are you using Firefox? It doesn't support [`Element.scrollIntoViewIfNeeded()`](https://caniuse.com/scrollintoviewifneeded). I think I can just replace it with `scrollIntoView` like in https://github.com/janosh/svelte-multiselect/issues/87 and release a new version of `svelte-toc`. Then to fix this, you just need to `yarn add -D svelte-toc` and commit the changes to `package.json` and `yarn.lock`.

**Comment by janosh** [comment]
I released `svelte-toc@0.2.10` in an attempt to fix this.

**Comment by dominikgeissler** [comment]
yup it works now

**Comment by janosh** [comment]
Glad to hear it.
